quantum/encoder.c

```diff
@@ -77,7 +77,23 @@ __attribute__((weak)) bool encoder_update_user(uint8_t index, bool clockwise) {
 }
 
 __attribute__((weak)) bool encoder_update_kb(uint8_t index, bool clockwise) {
-    return encoder_update_user(index, clockwise);
+    bool res = encoder_update_user(index, clockwise);
+    if (res) {
+        if (clockwise) {
+#ifdef EXTRAKEY_ENABLE
+            tap_code_delay(KC_VOLU, 10);
+#else
+            tap_code_delay(KC_PGDN, 10);
+#endif
```

Comment:
```suggestion
#if defined(EXTRAKEY_ENABLE)
            tap_code_delay(KC_VOLU, 10);
#elif defined(MOUSEKEY_ENABLE)
            tap_code_delay(KC_MS_WH_UP, 10);
#else
            tap_code_delay(KC_PGDN, 10);
#endif
```

Reply:
could probably also use `ENCODER_MAP_KEY_DELAY` instead of 10. but that gets set as 0 by default.